docs/install/ConfiguringOsmApiDatabaseConflation.asciidoc

```diff
@@ -0,0 +1,14 @@
+
+[[HootConfiguringOsmApiDatabaseConflation]]
+== Configuring OSM API Database Conflation
+
+By default, Hootenanny is configured with an empty test OSM API database.  To conflate against a production OSM API database:
+
+* Create a custom OSM API database configuration file $HOOT_HOME/conf/DatabaseConfigLocal.sh by copying the file 
+$HOOT_HOME/conf/DatabaseConfigDefault.sh
+* Fill in the following settings with your production OSM API database information:
+** database server name: DB_HOST_OSMAPI
+** database name: DB_NAME_OSMAPI
+** database port: DB_PORT_OSMAPI
+** database username for conflating data: DB_USER_OSMAPI
+** database password: DB_PASSWORD_OSMAPI

```

Comment:
Ok, got it.  I'm going to log an issue for that and put a relevant warning in the docs.  So far, the only place that might be using this with any non-test data may onsite and I don't think the vagrant instance is used there.  But still want to make sure the data stays safe regardless of how hoot is used.